src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -72,8 +73,14 @@ public enum OperationKind
         /// <summary>Indicates an <see cref="IEventReferenceOperation"/>.</summary>
         EventReference = 0x1e,
         /// <summary>Indicates an <see cref="IUnaryOperation"/>.</summary>
+        Unary = 0x1f,
+        /// <summary>Indicates an <see cref="IUnaryOperation"/>. Use <see cref="Unary"/> instead.</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Should those be marked as obsolete?

Reply:
Compat council did not want to mark these as obsolete to not break consumer builds.